docs/pages/index.tsx

```diff
@@ -20,7 +20,7 @@ const Home: NextPage = () => {
 
       <main>
         <h1 className="text-xl text-sky-500 md:text-2xl lg:text-3xl">Welcome to CooperTS</h1>
-
+        <div>A collection of methods and a way of programming that aims to reduce the cost of building correct software.</div>
```

Comment:
@SamEdwards-1 Do you have any input on styles here? Or more generally, how should we go about styling new pieces of the app?